Sources/MessagingInApp/MessagingInAppImplementation.swift

```diff
@@ -100,9 +95,7 @@ extension MessagingInAppImplementation: GistDelegate {
         // a close action does not count as a clicked action.
         if action != "gist://close" {
             if let deliveryId = getDeliveryId(from: message) {
-                // the state of the SDK does not change if adding this queue task isn't successful so ignore result
-                // FIXME: [CDP] Pass to Journey
-                // _ = queue.addTrackInAppDeliveryTask(deliveryId: deliveryId, event: .clicked, metaData: ["action_name": name, "action_value": action])
+                busEventManager.postEvent(TrackInAppMetricEvent(deliveryID: deliveryId, event: InAppMetric.clicked.rawValue, params: ["action_name": name, "action_value": action]))
```

Comment:
```suggestion
                busEventManager.postEvent(TrackInAppMetricEvent(deliveryID: deliveryId, event: InAppMetric.clicked.rawValue, actionName: name, actionValue: action)))
```

Could we avoid using hard-coded strings here?

Reply:
I am not sure i follow, are you suggesting adding these as part of event? but they are meta data, and they can be more of these in future, we shouldn't ideally be adding all of the meta data attributes as parameters? 

hence `param` acts as optional meta data for event